api/models/people.py

```diff
@@ -87,6 +88,26 @@ class Meta:
     def __str__(self) -> str:
         return f"{self.full_name} ({self.pk})"
 
+    def save(
+        self,
+        force_insert: bool = False,
+        force_update: bool = False,
+        using: str | None = None,
+        update_fields: Iterable[str] | None = None,
+    ) -> None:
+        def capitalize_each_word(str_: str) -> str:
+            return " ".join(part.capitalize() for part in str_.split())
+
+        self.first_name = capitalize_each_word(self.first_name)
+        self.last_name = capitalize_each_word(self.last_name)
+        self.email = self.email.casefold()
```

Comment:
Could you please explain, why not `.lower()`?

I've just known about such method existence, never seen it's usage before

Reply:
A good explanation is e.g. [here](https://stackoverflow.com/a/45745761). I actually think I should use `.lower()` in this case because we are persisting, not comparing. I was thinking of comparisons (data sent by bot vs. data already in database) and used `.casefold()`, but in the context of `.save()` it's probably wrong.